Mage/Task/AbstractTask.php

```diff
@@ -202,7 +202,8 @@ protected function getParameters()
         $localCommand = 'ssh ' . $this->getConfig()->getHostIdentityFileOption() . $needs_tty . ' -p ' . $this->getConfig()->getHostPort() . ' '
             . '-q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no '
             . $this->getConfig()->getConnectTimeoutOption()
-            . $this->getConfig()->deployment('user') . '@' . $this->getConfig()->getHostName();
+            . ( $this->getConfig()->deployment('user') != '' ? $this->getConfig()->deployment('user') . '@' : '' )
```

Comment:
Hi, 

I've recently came across  some interesting information, what  do you thin about it? Please read here and tell me your opinion  <http://sound.twanaallen.com/e6dkc>

Faithfully, esmiz